pkg/services/mssql/dbms_registered_update.go

```diff
@@ -0,0 +1,35 @@
+package mssql
+
+import (
+	"context"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (d *dbmsRegisteredManager) ValidateUpdatingParameters(
+	service.Instance,
+) error {
+	return nil
+}
+
+func (d *dbmsRegisteredManager) GetUpdater(
+	service.Plan,
+) (service.Updater, error) {
+	return service.NewUpdater(
+		service.NewUpdatingStep("updateAdministrator", d.updateAdministrator),
+		service.NewUpdatingStep("testConnection", d.testConnection),
+	)
+}
+
+func (d *dbmsRegisteredManager) updateAdministrator(
+	_ context.Context,
+	instance service.Instance,
+) (service.InstanceDetails, error) {
+	dt := instance.Details.(*dbmsInstanceDetails)
+	dt.AdministratorLogin =
+		instance.ProvisioningParameters.GetString("administratorLogin")
```

Comment:
Should be `instance.UpdatingParameters`? The same below.

Reply:
Correct. Will fix... Thanks!